content/docs/concepts/components/stack.md

```diff
@@ -6,7 +6,48 @@ aliases=[
 ]
 +++
 
-## What is a stack?
+## What is a stack and Why is it deprecated?
+
+### Deprecation
+Stacks are great - long live the stack! 
+However, a stack is basically an extra layer on top of the **build image** (where your app is built)
+and the **run image** (where your app runs) and [the community decided](https://github.com/buildpacks/rfcs/blob/main/text/0096-remove-stacks-mixins.md)
+to use those two concepts directly without the additional layer formerly known as _stack_. 
+
+Stacks will continue to be supported with full backwards compatibility.
+
+### What's Next?
+[Targets](https://github.com/buildpacks/rfcs/blob/main/text/0096-remove-stacks-mixins.md#example-buildpacktoml-targets-table) replace stacks.
+Targets allow buildpack authors to directly specify details such as OS and architecture directly without the intermediate object known as a "stack." 
+Targets are available starting with the 0.12 Platform API but should be safely ignored by older platforms.
+
+
+#### Wait - are you cutting me off?
+My friend, never fear. We have a long horizon and a heartfelt desire to ease your transition. We will provide backwards compatibility with stacks for so long that we haven't even decided yet when we're going to turn them off.
```

Comment:
I like the more personal style that you use.  However we're much more straight-laced in the rest of the docs.  Should we prefer the (more boring) existing style?

```suggestion
My friend, never fear. We have a long horizon and a heartfelt desire to ease your transition. We will provide backwards compatibility with stacks for so long that we haven't even decided yet when we're going to turn them off.
```

Reply:
oh yes, almost certainly.